content/applications/sales/point_of_sale/pricing/digital_labels.rst

```diff
@@ -0,0 +1,151 @@
+==============
+Digital labels
+==============
+
+Digital labels allow you to display important product information like product prices and barcodes
+on store shelves and to update them remotely from the backend, eliminating the need to print new
+labels when product information changes.
+
+.. image:: digital_labels/electronic-label.png
+   :alt: electronic label from Pricer
+
+.. note::
+   Odoo uses digital labels from `Pricer <https://www.pricer.com/>`_.
+
+Configuration
+=============
+
+Pricer setup
+------------
+
+#. `Get in touch with Pricer <https://www.pricer.com/contact>`_ to create and configure your Pricer
+   account.
+#. Create your stores: one pricer store equates to one physical store.
+#. Create the following variables to allow sharing of information between Odoo and Pricer:
+
+   - `itemId` : the internal ID of your products
+   - `itemName` : the name of your products
+   - `price` : the price of your products, taxes included
+   - `presentation`: the template name used for Pricer tags display
```

Comment:
I've changed 3 to try and make this clearer. There's no example because itemID should = itemID x) but maybe it's clearer this way? (I've had to ask for an explanation bc I, too had a hard time understanding)